Homework_1_User_input/Homework_1_User_input/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace Homework_1_User_input
+{
+    class Program
+    {
+       
+        static void Main(string[] args)
+        {
+            for (int i = 0; i<=1;) { 
+            Console.WriteLine("Please enter name: ");
+            string name = Console.ReadLine();
+
+            Console.WriteLine("Please enter surname: ");
+            string surname = Console.ReadLine();
+
+            Console.WriteLine("Please enter age: ");
+            string ageText = Console.ReadLine();
+            int age = int.Parse(ageText);
+
+            Console.WriteLine("Please enter kg: ");
+            string kgText = Console.ReadLine();
+            double kg = double.Parse(kgText);
+
+            Console.WriteLine("Please enter height: ");
+            string heightText = Console.ReadLine();
```

Comment:
These two lines could be reuse through a function `Prompt`- which asks to input and reads it.